controllers/dscinitialization/servicemesh_setup.go

```diff
@@ -88,7 +88,7 @@ func configureServiceMeshFeatures() feature.FeaturesProvider {
 
 		cfgMapErr := feature.CreateFeature("mesh-shared-configmap").
 			For(handler).
-			WithResources(servicemesh.ConfigMaps).
+			WithResources(servicemesh.MeshRefs, servicemesh.AuthRefs).
```

Comment:
I hope this resolves Wen feedback in a previos PR about that this `ConfigMaps` wasn't really a good name :-)